api/app/src/providers/oura.ts

```diff
@@ -61,6 +61,10 @@ export class Oura extends Provider implements OAuth2 {
     return this.oauth.getTokenFromAuthCode(code);
   }
 
+  async revokeProviderAccess(connectedUser: ConnectedUser) {
+    return this.oauth.revokeWithNoOauth(connectedUser);
```

Comment:
Just to confirm, Oura and Whoop should also use the `revokeWithNoOauth`?

Reply:
Correct yea